client/src/components/dashboard/PositionCard.js

```diff
@@ -53,14 +54,14 @@ const PositionCard = (props) => {
       <NotationDisplay
         open={open}
         handleClose={handleClose}
-        name={name}
+        name={nameToUse}
```

Comment:
Hmm, I thought what we discussed was to only show the base opening name card. And then when we click on the base opening card, we'll show a dropdown of variations or something.